### PR TITLE
Couchbase Alerts

### PR DESCRIPTION
**Pull request by PhuNguyenMTT**

### CODE REVIEW

**Review by arunpatyal — CHANGES_REQUESTED** [review]

File: monitor_packages/Couchbase/couchbase.json [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+          "occurrenceType": "Always"
+        }
+      ],
+      "notifications": [],
+      "isDisabled": true,
+      "groupNotifications": true,
+      "playbook": ""
+    },
+    {
+      "name": "Couchbase - Too Many Login Failure",
```
[the remark this answers is not in the document]
Comment (PhuNguyenMTT) [reply monitor_packages/Couchbase/couchbase.json]: updated

File: monitor_packages/Couchbase/couchbase.tf [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+				trigger_source = "AnyTimeSeries"
+				trigger_type = "ResolvedCritical",
+				detection_method = "MetricsStaticCondition"
+			  }
+			]
+}
+module "Couchbase-TooManyLoginFailure" {
+  source                    = "SumoLogic/sumo-logic-monitor/sumologic"
+  #version                  = "{revision}"
+  monitor_name                = "Couchbase - Too Many Login Failure"
```
[the remark this answers is not in the document]
Comment (PhuNguyenMTT) [reply monitor_packages/Couchbase/couchbase.tf]: updated

**Comment by arunpatyal** [comment]
We are using the same variable for critical, warning and missing data, this generates error if customer specifies critical connection or email action. Need to make changes as per this [PR](https://github.com/SumoLogic/terraform-sumologic-sumo-logic-monitor/pull/49).

**Comment by PhuNguyenMTT** [comment]
i updated for variables for critical, warning and missing data

**Pull request merged**